### ISSUE TITLE
tsc semantic error

### ISSUE THREAD

**Issue by len-ro** [aurelia/i18n#186]
**I'm submitting a bug report**

* **Library Version:**
1.1.2

* **Operating System:**
Linux (Ubuntu 16.06)

* **Node Version:**
6.2.0

* **NPM Version:**
3.10.6

* **JSPM Version**
JSPM 0.16.46

* **Browser:**
al

* **Language:**
TypeScript 2.2

**Current behavior:**

I am trying to get rid of all typescript semantic errors in a project based on aurelia typescript skeleton. One of the errors remaining is:

`typings/modules/aurelia-i18n/index.d.ts(4,8): error TS1192: Module ''i18next'' has no default export.`

This is due to the line

`import i18next from 'i18next';`

since the i18next module does not have a default export. Just removing the line fixes the error for me and seems to have no side effect. Maybe this change can be applied to git version also.

Thanks.

**Comment by zewa666** [comment]
You should take a read at the [official i18n docs](http://aurelia.io/hub.html#/doc/article/aurelia/i18n/latest/i18n-with-aurelia/4). This is a known issue and results from the way how TypeScript handles synth default imports.

TL;DR use `"allowSyntheticDefaultImports": true` in your tsconfig and everything should be fine.

**Comment by len-ro** [comment]
Thank you, haven't noticed the doc moved from git.
